src/Mod/Path/App/Path.cpp

```diff
@@ -434,7 +435,7 @@ void Toolpath::recalculate(void) // recalculates the path cache
 
 unsigned int Toolpath::getMemSize (void) const
 {
-    return toGCode().size();
+    return vpcCommands.size() * 100;
```

Comment:
This seems arbitrary -- why 100?

Reply:
The purpose of `getMemSize()` is to get an estimate of the memory required for saving the object. It is used when copying objects, and FC use it to decide whether to make the copy in memory or using a temporary file. Here I just use 100 as a rough estimation of a single command size.